packages/grafana-ui/src/components/Cascader/Cascader.tsx

```diff
@@ -62,6 +63,7 @@ export class Cascader extends React.PureComponent<CascaderProps, CascaderState>
       cpy.push(option);
       if (!option.items) {
         selectOptions.push({
+          singleLabel: cpy[cpy.length - 1].label,
```

Comment:
What is the singleLabel?

Reply:
It is the label for only the value rather than the path. Should probably be renamed to path and label instead.

Reply:
Oh, never mind. `label` is required by react-select when searching.